xlml/utils/name_format.py

```diff
@@ -33,22 +33,35 @@ def generate_run_name(benchmark_id: str) -> str:
 
 
 @task
-def generate_tb_file_location(run_name: str, base_output_directory: str) -> str:
+def generate_tb_file_location(
```

Comment:
Probably we missed this before. It looks like we should enable a path `<base_output_directory>/whatever_folder/events.out.tfevents.*`. Otherwise, we could nested more and more different formats. No need to address in this PR, but just call out. cc @richardsliu

Reply:
Ack, agree it'd be good to have a more generic version of this function.